README.md

```diff
@@ -94,6 +107,31 @@ This plugin also supports flight recording (see next section)
 --tf-replay=[record|replay|disable]
 ```
 
+### Teardown Options
+
+`pytest_terraform` supports three different teardown modes for the terraform decorator.
+The default, `pytest_terraform.options.teardown.ON` will always attempt to teardown any and all modules via `terraform destory`.
```

Comment:
can we hoist the import path to pytest_terraform.teardown in the package __init__ and refer to the shortened form in docs.